internal/config/config.go

```diff
@@ -34,25 +34,31 @@ type Default struct {
 
 	// TZ represent system time location .
 	TZ string `json:"time_zone" yaml:"time_zone"`
+
+	// Log represent log configuration.
+	Log `json:"log" yaml:"log"`
```

Comment:
please do not embedd struct in this case

Reply:
thank you for your review!
I will fix soon.